src/internal/connector/graph/cache_container.go

```diff
@@ -169,20 +169,21 @@ func CreateCalendarDisplayable(entry any, parentID string) *CalendarDisplayable
 // helper funcs
 // =========================================
 
-// checkRequiredValues is a helper function to ensure that
-// all the pointers are set prior to being called.
-func CheckRequiredValues(c Container) error {
-	id, ok := ptr.ValOK(c.GetId())
-	if !ok {
-		return clues.New("container missing ID")
+// CheckIDAndName is a validator that ensures the ID
+// and name are populated and not zero valued.
+func CheckIDAndName(c Container) error {
+	if c == nil {
+		return clues.New("nil container")
 	}
 
-	if _, ok := ptr.ValOK(c.GetDisplayName()); !ok {
-		return clues.New("container missing display name").With("container_id", id)
+	id := ptr.Val(c.GetId())
+	if len(id) == 0 {
+		return clues.New("container missing ID")
 	}
 
-	if _, ok := ptr.ValOK(c.GetParentFolderId()); !ok {
```

Comment:
Was removal of `GetParentForlderId` intentional?

Reply:
we should definitely have a function that checks for ID, display name, and parent ID (it can call this if we want)

Reply:
Ah, weird diff.  `CheckRequiredValues` wasn't getting called anywhere, so we've never actually checked for the parentID, making this a non-change.  It just looks like I removed it because I moved `CheckIDAndName`, which _is_ getting called, into this file,